### PR TITLE
11538 - Allow for multiple name request query.

### PR DESCRIPTION
**Pull request by seeker25**
*Issue #:* [/bcgov/entity/#11538](https://github.com/bcgov/entity/issues/11538)

*Description of changes:*
Tweak to allow multiple NRs to be requested at once.

Related PR for Namex: https://github.com/bcgov/namex/pull/1237

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Comment by seeker25** [comment]
Please don't merge, going to test this after some high level approval.

**Review by pwei1018 — APPROVED** [review]

**Pull request closed without merging**
